examples/chart/teleport-cluster/templates/proxy/service.yaml

```diff
@@ -29,7 +29,7 @@ spec:
   {{- toYaml . | nindent 2 }}
 {{- end }}
   ports:
-  - name: https
+  - name: tls
```

Comment:
Isn't this potentially going to break some existing setups?

Reply:
It shouldn't do - it's more of an internal name which refers to the port which is exposed. If people are using their own service to expose Teleport they tend to redeploy it with their own name/config etc.

We're already making a lot of migrations/changes as part of the v12 Helm chart upgrade - I can make sure to add something to the release notes which explains that the port has been renamed.